client/src/components/embedding/components/Opacities/components/Opacity/index.tsx

```diff
@@ -30,10 +28,7 @@ const MARKS = [
 interface Props {
   name: string;
   value: number;
-  handleChange: (
-    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
-    value: number | number[]
-  ) => void;
+  handleChange: (event: Event, value: number | number[]) => void;
```

Comment:
I think this change should resolve the `handleChange(event as unknown as Event,` below!

```suggestion
  handleChange: (
    event: Event | React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
    value: number | number[]
  ) => void;
```